src/templates/specimen/body/annotate/annotationTypes/Correcting.js

```diff
@@ -0,0 +1,14 @@
+import { Row, Col } from 'react-bootstrap';
+
+
+const Correcting = () => {
+    return (
+        <Row>
+            <Col>
+                Test
```

Comment:
Lijkt op een overblijfsel van iets. Of iets wat nog moet komen. Geen probleem maar als hij overbodig is misschien verwijderen

Reply:
Deze file wordt inderdaad vervangen in de nieuwe versie, splits daarin de annotatie types op de form variant (voor het invullen van de gegevens) en de weergave voor in het linker vlak (ipv de voormalige chat)